### ISSUE TITLE
SXD: OPI for SXD Jaws Mirror

### ISSUE THREAD

**Issue by kjwoodsISIS** [ISISComputingGroup/IBEX#6058]
As a SXD scientist I want to be able to control the jaws mirror, so that I can control whether it is in/out of the beam.

#### Acceptance Criteria
1. Using the IBEX client, I can control the Jaws Mirror (IN/OUT positions)
1. Using genie_python, I can control the Jaws Mirror (IN/OUT positions)

#### Notes
1. Check the [SXD Instrument wiki](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/SXD-Instrument-Details).
1. The SXD Jaws Mirror is related to the SXD Jaws VI.
1. The SXD Jaws Mirror is controlled by LinMot drives
1. The OPI must offer functionality equivalent to the existing VI (located here: `C:\LabVIEW Modules\Instruments\SXD\SXD Jaws` - specifically the `SXD Jaws (Mirror) Driver.vi` file in the `Screens` sub-directory).
1. **Note from Instrument Scientist:** 
   * The mirror on SXD is located upstream from the jaws. Its purpose is to direct laser light from an external laser to the sample position.  The reason for the mirror is that we cannot have the laser in the vacuum of the flight path, but we can have a mirror. Most of the time it is parked on the side away from the direct beam.

**Comment by LowriJenkins** [comment]
In/Out motion setpoints setup, will probably still need an OPI but we should make a generic one.

**Comment by danielmaclaren** [comment]
http://control-svcs.isis.cclrc.ac.uk/git/?p=instconfigs/inst.git;a=shortlog;h=refs/heads/ticket_6058_sxd

For the OPI, we will use the motionsetpoint(few).

**Comment by danielmaclaren** [comment]
Tested with a simulated LINMOT, so reviewer should just merge our branch into NDXSXD.
